themes/qgis-theme/layout.html

```diff
@@ -24,6 +24,12 @@
   {%- set titlesuffix = "" %}
 {%- endif %}
 
+{%- macro outdated() %}
+    <div class="row outdated">
+      This documentation is for a QGIS version which has reached end of life. Visit the <a href="https://www.qgis.org/en/docs/index.html">latest version</a> instead.
```

Comment:
```suggestion
      This documentation is for a QGIS version which has reached end of life. Visit the <a href="https://docs.qgis.org/latest/">latest version</a> instead.
```
which will currently bring to https://docs.qgis.org/3.4/en/docs/ (since the latest redirection is already in place)